UefiTestingPkg/FunctionalSystemTests/SmmPagingProtectionsTest/Smm/SmmPagingProtectionsTestSmm.c

```diff
@@ -12,13 +12,19 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #include <PiSmm.h>
 
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/DebugLib.h>
+#include <Library/IoLib.h>
+#include <Library/MemoryAllocationLib.h>
 #include <Library/SmmServicesTableLib.h>
 #include <Library/SmmMemLib.h>
-#include <Library/DebugLib.h>
-#include <Library/BaseMemoryLib.h>
 
 #include <Protocol/SmmExceptionTestProtocol.h>
 
+#include <Register/Intel/ArchitecturalMsr.h>
```

Comment:
This is probably a good indicator that we need a library/abstraction so that our test can apply cross silicon.